Generate_Client/README.md

```diff
@@ -0,0 +1 @@
+Do not edit the config.json file
```

Comment:
It would be nice to have some basic instructions on how to run the generator script.

Reply:
Ack. I had written the instructions on how to run the generator script on docs/client_library_generation.md(which was linked from python-gluster-mgmt-client/README.md) . Will move it to Generate_Client/ and delete the existing README.md.